acre/core.py

```diff
@@ -66,13 +69,17 @@ def compute(env,
     # Format dynamic values
     for key in reversed(result.sorted):
         if key in env:
+            if not isinstance(env[key], str):
+                continue
```

Comment:
What kind of values are you expected in the input `env`. I'd say that anything that is not a string is not an "environment value" and thus is invalid input. That should not be continued but **should** raise an error. It means the input value is wrong.

Unless maybe we'd want to explicitly skip None values?